sections/sponsor-info/dist/sponsor-info.module.css

```diff
@@ -0,0 +1,111 @@
+.sponsor-info {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  font-size: 18px;
+  padding-top: 40px;
+  padding-bottom: 40px;
+  background-color: #f3f4f5;
+}
+@media (min-width: 0em) {
```

Comment:
any reason this is a css file and not scss? was wondering why we aren't using the breakpoint mixins here 😂

Reply:
It somehow got built into the /dist folder lol